gossip/src/gossip_service.rs

```diff
@@ -193,6 +193,7 @@ pub fn discover(
 }
 
 /// Creates a ThinClient by selecting a valid node at random
+#[deprecated(since = "1.18.0", note = "Interface will change")]
```

Comment:
Oh shoot, I didn't look carefully enough at this, sorry. Can you please fix this on master too?
```suggestion
#[deprecated(since = "1.18.6", note = "Interface will change")]
```